schema/constants/matte-mode.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "integer",
+    "title": "Matte Mode",
+    "description": "How a layer should mask another layer",
+    "oneOf": [
+        {
+            "title": "Normal",
+            "description": "The layer is not used as a track matte",
+            "const": 0
+        },
+        {
+            "title": "Alpha",
+            "description": "The layer's opacity will affect the masked layer (showing opaque areas)",
```

Comment:
> showing opaque areas

This could be misread as only opaque/transparent matte areas being shown.  Not sure how to put in in both readable and exact wording, maybe something along these lines:


*The masked layer opacity is modulated by the track matte layer opacity.*

*The masked layer opacity is modulated by the inverted track matte layer opacity.*

*The masked layer opacity is modulated by the track matte layer luminance.*

*The masked layer opacity is modulated by the inverted track matte layer luminance.*

?